starfish/image/_filter/richardson_lucy_deconvolution.py

```diff
@@ -45,14 +45,14 @@ def add_arguments(cls, group_parser: argparse.ArgumentParser) -> None:
         group_parser.add_argument(
             '--sigma', type=float, help='standard deviation of gaussian kernel')
         group_parser.add_argument(
-            '--clip', action='store_true',
-            help='(default False) if True, clip values below -1 and above 1')
+            '--no-clip', action='store_false',
+            help='(default True) if True, clip values below 0 and above 1')
 
     # Here be dragons. This algorithm had a bug, but the results looked nice. Now we've "fixed" it
     # and the results look bad. #548 addresses this problem.
     @staticmethod
     def richardson_lucy_deconv(
-            image: np.ndarray, iterations: int, psf: np.ndarray, clip: bool=False) -> np.ndarray:
```

Comment:
Since this method is basically internal to this class, I think we should just make the clip parameter required (i.e., no default value).

Reply:
Also, docblock below needs to be updated.